test/test_xcpp_kernel.py

```diff
@@ -131,6 +131,13 @@ class XCppTests(jupyter_kernel_test.KernelTests):
                 'mime': 'image/png'
             }
         ]
+        code_omp='#include <iostream>\n#include "omp.h"\n#include "clang/Interpreter/CppInterOp.h"\nCpp::LoadLibrary("libomp");omp_set_thread_num(4);#pragma omp parallel\n{\nif(omp_get_thread_num()==2) std::cout<<omp_get_thread_num()<<std::endl;}'
+        def test_xcpp_omp(self):
+            self.flush_channels()
+            reply, output_msgs = self.execute_helper(code=self.code_omp,timeout=100)
```

Comment:
Try to dump reply and output_msgs as string.

Reply:
I now have it running without timeout. It says it cannot find the omp header.

Reply:
@vgvassilev @alexander-penev 
The error after I managed to avoid the timeout issue is
input_line_11:2:10: fatal error: 'omp' file not found
E       -     2 | #include <omp>
E       -       |          ^~~~~
E       - Failed to parse via ::process:Parsing failed.

Reply:
We seem to have failed to pass the relevant `-I` or we did not install openmp?

Reply:
The issue looks like its with the line #pragma omp parallel, but unsure what exactly it doesn't seem to like.